pkg/cmd/create/install.go

```diff
@@ -2308,10 +2308,10 @@ func (options *InstallOptions) createSystemVault(client kubernetes.Interface, na
 			CreateOptions: CreateOptions{
 				CommonOptions: options.CommonOptions,
 			},
-			IngressConfig:       *ic,
```

Comment:
Why is ingress config and namespace removed from the arguments? I think in the installed was just used to expose vault with the ingress configuration.

Reply:
I removed them because there were basically not being used at all.

Reply:
It makes sense then. Thanks!